python/cugraph/cugraph/testing/__init__.py

```diff
@@ -11,7 +11,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cugraph.testing.utils import RAPIDS_DATASET_ROOT_DIR_PATH, RAPIDS_DATASET_ROOT_DIR
+from cugraph.testing.utils import (
+    RAPIDS_DATASET_ROOT_DIR_PATH,
+    RAPIDS_DATASET_ROOT_DIR,
+    RAPIDS_RESULTSET_ROOT_DIR_PATH,
```

Comment:
I think it might be better to start off by just using `RAPIDS_DATASET_ROOT_DIR`, then adding another env var just for Resultset only if we realize it's needed.

Reply:
Sure, I removed `RAPIDS_RESULTSET_ROOT_DIR_PATH`.